.publicApi/Microsoft.ApplicationInsights.dll/netstandard2.0/PublicAPI.Shipped.txt

```diff
@@ -404,490 +156,51 @@ Microsoft.ApplicationInsights.DataContracts.SeverityLevel.Warning = 2 -> Microso
-Microsoft.ApplicationInsights.Extensibility.Implementation.HttpWebResponseWrapper.RetryAfterHeader.get -> string
-Microsoft.ApplicationInsights.Extensibility.Implementation.HttpWebResponseWrapper.RetryAfterHeader.set -> void
-Microsoft.ApplicationInsights.Extensibility.Implementation.HttpWebResponseWrapper.StatusCode.get -> int
-Microsoft.ApplicationInsights.Extensibility.Implementation.HttpWebResponseWrapper.StatusCode.set -> void
-Microsoft.ApplicationInsights.Extensibility.Implementation.HttpWebResponseWrapper.StatusDescription.get -> string
-Microsoft.ApplicationInsights.Extensibility.Implementation.HttpWebResponseWrapper.StatusDescription.set -> void
-Microsoft.ApplicationInsights.Extensibility.Implementation.InternalContext
-Microsoft.ApplicationInsights.Extensibility.Implementation.InternalContext.AgentVersion.get -> string
-Microsoft.ApplicationInsights.Extensibility.Implementation.InternalContext.AgentVersion.set -> void
-Microsoft.ApplicationInsights.Extensibility.Implementation.InternalContext.NodeName.get -> string
-Microsoft.ApplicationInsights.Extensibility.Implementation.InternalContext.NodeName.set -> void
-Microsoft.ApplicationInsights.Extensibility.Implementation.InternalContext.SdkVersion.get -> string
-Microsoft.ApplicationInsights.Extensibility.Implementation.InternalContext.SdkVersion.set -> void
-Microsoft.ApplicationInsights.Extensibility.Implementation.JsonSerializer
-Microsoft.ApplicationInsights.Extensibility.Implementation.LocationContext
-Microsoft.ApplicationInsights.Extensibility.Implementation.LocationContext.Ip.get -> string
-Microsoft.ApplicationInsights.Extensibility.Implementation.LocationContext.Ip.set -> void
-Microsoft.ApplicationInsights.Extensibility.Implementation.OperationContext
-Microsoft.ApplicationInsights.Extensibility.Implementation.OperationContext.CorrelationVector.get -> string
-Microsoft.ApplicationInsights.Extensibility.Implementation.OperationContext.CorrelationVector.set -> void
-Microsoft.ApplicationInsights.Extensibility.Implementation.OperationContext.Id.get -> string
-Microsoft.ApplicationInsights.Extensibility.Implementation.OperationContext.Id.set -> void
-Microsoft.ApplicationInsights.Extensibility.Implementation.OperationContext.Name.get -> string
-Microsoft.ApplicationInsights.Extensibility.Implementation.OperationContext.Name.set -> void
-Microsoft.ApplicationInsights.Extensibility.Implementation.OperationContext.ParentId.get -> string
-Microsoft.ApplicationInsights.Extensibility.Implementation.OperationContext.ParentId.set -> void
-Microsoft.ApplicationInsights.Extensibility.Implementation.OperationContext.SyntheticSource.get -> string
-Microsoft.ApplicationInsights.Extensibility.Implementation.OperationContext.SyntheticSource.set -> void
+Microsoft.ApplicationInsights.DataContracts.TraceTelemetry.TraceTelemetry(string message, Microsoft.ApplicationInsights.DataContracts.SeverityLevel severityLevel) -> void
```

Comment:
We shouldn't introduce any new APIs. Change shows that there are 2 modifications apart from deletion, can you check?